docs/README.rst

```diff
@@ -118,7 +118,7 @@ Define the parameters in the ``env`` section of the `Controller application mani
    +----+-------------------------------------+---------+----------+----------------+---------------------------------------------------------------------------------+----------------------+
    |    | external_addr [#extaddr]_           | string  | Required | n/a            | Virtual address on the BIG-IP to use for cloud ingress.                         |                      |
    +----+-------------------------------------+---------+----------+----------------+---------------------------------------------------------------------------------+----------------------+
-   |    | tier2_ip_range                      | string  | Required | n/a            | IP range to assign to the tier2 vips (required in Service Broker mode only)     | Must use CIDR        |
+   |    | tier2_ip_range                      | string  | Optional | 172.0.0.1/24   | IP range to assign to the tier2 vips (required in Service Broker mode only)     | Must use CIDR        |
```

Comment:
Update the default to match what's in the code.

Reply:
Good catch thx!